bundles/org.openhab.binding.metofficedatahub/src/main/resources/OH-INF/thing/thing-types.xml

```diff
@@ -0,0 +1,202 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<thing:thing-descriptions bindingId="metofficedatahub"
+	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xmlns:thing="https://openhab.org/schemas/thing-description/v1.0.0"
+	xsi:schemaLocation="https://openhab.org/schemas/thing-description/v1.0.0 https://openhab.org/schemas/thing-description-1.0.0.xsd">
+
+	<bridge-type id="bridge">
+
+		<label>MetOffice DataHub Bridge</label>
+		<description>MetOffice DataHub API Bridge</description>
+
+		<properties>
+			<property name="Site Specific API Call Count"/>
+		</properties>
+
+		<config-description>
+			<parameter name="siteSpecificClientId" type="text" required="true">
+				<context>password</context>
```

Comment:
ClientId is usually not considered a secret so i think the context tag should not be used.

Reply:
This will be removed I need to update the binding to use a single parameter instead of the two they had. That will be a API key, which - I have not checked but looks to be a base64 encoded value. As its the authentication key for the API's I'll keep that one as password. This one and the other one will go so I think that should resolve this one.